pulpito/templates/run.html

```diff
@@ -52,6 +52,9 @@ <h2>
               <h2>
                 <a href="/{{ run.name }}/detail"> See detail view?</a>
               </h2>
+              <h2>
+                <a href="http://qa-proxy.ceph.com/{{ run.name }}"> Want to see scrape.log?</a>
+              </h2>
```

Comment:
Should be
```suggestion
              <h2>
                <a href="http://qa-proxy.ceph.com/teuthology/{{ run.name }}/scrape.log"> Want to see scrape.log?</a>
              </h2>
```

Reply:
sure ! i check

Reply:
@hkaur008 @kamoltat any updates regarding that PR?